cartesian_traject_controller:
    type: franka_example_controllers/CartesianTrajectoryController
    arm_id: $(arg arm_id)
    joint_names:
        - $(arg arm_id)_joint1
        - $(arg arm_id)_joint2
        - $(arg arm_id)_joint3
        - $(arg arm_id)_joint4
        - $(arg arm_id)_joint5
        - $(arg arm_id)_joint6
        - $(arg arm_id)_joint7        

shy_controller:
    type: franka_example_controllers/ShyController
    arm_id: $(arg arm_id)
    joint_names:
        - $(arg arm_id)_joint1
        - $(arg arm_id)_joint2
        - $(arg arm_id)_joint3
        - $(arg arm_id)_joint4
        - $(arg arm_id)_joint5
        - $(arg arm_id)_joint6
        - $(arg arm_id)_joint7
    k_gains:
        - 400.0
        - 400.0
        - 400.0
        - 400.0
        - 200.0
        - 130.0
        - 35.0
    d_gains:
        - 40.0
        - 40.0
        - 40.0
        - 16.0
        - 16.0
        - 16.0
        - 8.0
    coriolis_factor: 1.0
    trajectory_deformed_length: 10
    admittance: 0.001